src/analysis/souffle/sensitivity_analysis.dl

```diff
@@ -0,0 +1,132 @@
+#ifndef SENSITIVITY_ANALYSIS_DL_
+#define SENSITIVITY_ANALYSIS_DL_
+//this file defines the calculations for sensitivity analysis 
+//to be used in dp calculations
+
+#include "src/analysis/souffle/taint.dl"
+
+//one isSqlOperator fact per each sql operator, currently supporting one output
+//referring to results as 'value' (sqlValue declared below) as per the discussion in the doc 
+.decl isSqlOperator(sqlOperator: Operation)
+isSqlOperator([owner, operator, result, input_list, attribute_list]) :- isOperation([owner, operator, result, input_list, attribute_list]), match("sql.*", operator). 
```

Comment:
We have [some handy relations](https://github.com/google-research/raksha/blob/main/src/analysis/souffle/operations.dl#L52) for getting fields of `Operation`s that should allow you to not fully deconstruct the `Operation` record type.

Reply:
I am not sure I understand what that would be useful for specifically.

Reply:
Got it now. Updated!